From 15bb2c2c640a89dfc42c3fa14d79e643192b13be Mon Sep 17 00:00:00 2001
From: Roberto Pereira <rpere@google.com>
Date: Thu, 23 Aug 2018 15:44:56 -0700
Subject: [PATCH] trusty_keymaster_ipc: Use ALOGV instead of ALOGE for info/debug message

This message was originally ALOGV but got accidentally changed during a
refactoring in 81ebcb1943850203c8feb261bf5737581f1b3fed

Bug:110153632
Test: VtsHalKeymasterV3_0TargetTest
Change-Id: Ibdfa0ab50cb8544c6f23c15049904f1741769647
Tracked-On: OAM-68619
Signed-off-by: Roberto Pereira <rpere@google.com>
Reviewed-on: 645063
---

diff --git a/trusty/keymaster/ipc/trusty_keymaster_ipc.cpp b/trusty/keymaster/ipc/trusty_keymaster_ipc.cpp
index 8c5cff6..0956fe6 100644
--- a/trusty/keymaster/ipc/trusty_keymaster_ipc.cpp
+++ b/trusty/keymaster/ipc/trusty_keymaster_ipc.cpp
@@ -167,7 +167,7 @@
         // TODO(swillden): Distinguish permanent from transient errors and set error_ appropriately.
         return translate_error(rc);
     } else {
-        ALOGE("Received %d byte response\n", rsp_size);
+        ALOGV("Received %d byte response\n", rsp_size);
     }
 
     const uint8_t* p = recv_buf;
